### DISCUSSION TITLE
How to make form update while file uploading with progressbar?

### DISCUSSION THREAD

**Discussion by qweluke** [react-hook-form/react-hook-form#3143] [Q&A]
I am trying to make a form with file upload progressbar but for some reason, the last element/file is not being updated.

What I am doing wrong?

Just select 2 or more files and check last element, especially text called "uploaded"
https://codesandbox.io/s/react-hook-form-usefieldarray-forked-m4vgq

What I want to achive:
- User select 1 or more files
- file preview appears on the page
- progress bar with % will show (already have that with axios)
- once the file is uploaded progressbar is hidden

**Comment by bluebill1049** [comment]
what's the issue in here? i find hard to understand the problem.
